docs/dev/parallel.zh.md

```diff
@@ -177,25 +159,18 @@ try {
 </div>
 
 
-### System Level
+### 系统层次
 
-A system-wide default parallelism for all execution environments can be defined by setting the
-`parallelism.default` property in `./conf/flink-conf.yaml`. See the
-[Configuration]({{ site.baseurl }}/ops/config.html) documentation for details.
+可以通过设置 ./conf/flink-conf.yaml 文件中的 parallelism.default 参数，在系统层次来指定所有执行环境的默认 parallelism。你可以通过查阅 [配置文档]({{ site.baseurl }}/zh/ops/config.html) 获取更多细节。
+获取更多细节。
 
-## Setting the Maximum Parallelism
+## 设置最大并发度
 
-The maximum parallelism can be set in places where you can also set a parallelism
-(except client level and system level). Instead of calling `setParallelism()` you call
-`setMaxParallelism()` to set the maximum parallelism.
+最大 parallelism 可以在所有设置 parallelism 的地方进行设定（除了客户端和系统层次）。与调用 `setParallelism()` 方法修改并发度相似，你可以通过调用 `setMaxParallelism()` 方法来设定最大 parallelism。
 
-The default setting for the maximum parallelism is roughly `operatorParallelism + (operatorParallelism / 2)` with
-a lower bound of `127` and an upper bound of `32768`.
+默认的最大 parallelism 大致等于算子的 parallelism + 算子的 parallelism/2，其下限为 127，上限为 32768。
```

Comment:
This sentence can be improved, please ref the [code](https://github.com/apache/flink/blob/6d7a15e328e8af6f001d6191c3f167bd8038bf14/flink-runtime/src/main/java/org/apache/flink/runtime/state/KeyGroupRangeAssignment.java#L127) for more information

the lower bound is `128` instead of `127`, please update the English doc also.

Reply:
ok